README.md

```diff
@@ -1,3 +1,84 @@
 # LOCK API
 
-API for LOCK.space international staking service
\ No newline at end of file
+API for LOCK.space international staking service
+
+# Documentation
+
+- [Registration](#registration)
+- [KYC](#kyc)
+- [Staking](#staking)
+- [Deposit](#deposit)
+- [Withdrawal](#withdrawal)
+- [Analytics](#analytics)
+- [Voting](#voting)
+- [Integration Example](#integration-example)
+
+# Registration
+
+1. Contact [support](support@lock.space) to registrate your wallet name
+2. Create an address on selected blockchain (e.g. DeFiChain)
+3. Get and sign message from sign-message endpoint: https://dev.api.lock.space/swagger/#/Authentication/AuthController_getSignMessage
+4. Register the user with the sign-up endpoint: https://dev.api.lock.space/swagger/#/Authentication/AuthController_signUp
+5. Now you can get your access token (with address & signature) with the sign-in endpoint: https://dev.api.lock.space/swagger/#/Authentication/AuthController_signIn
+
+# KYC
+
+Before the user can stake he has to proceed a KYC.
+
+1.  Trigger the kyc endpoint for KYC registration: https://dev.api.lock.space/swagger/#/KYC/KycController_startKyc
+
+2.  User has to do KYC
+
+        a) If the address is already registered and the user processed the KYC process at DFX.swiss you can transfer the KYC data with the transfer endpoint at DFX API:
+
+    https://api.dfx.swiss/swagger/#/auth/AuthController_signIn
+    https://api.dfx.swiss/swagger/#/kyc/KycController_transferKycData
+
+        b) Get and open the KYC Link which you can get from kyc endpoint after register KYC or from user endpoint ("kycLink"):
```

Comment:
If not, the user can do KYC at LOCK. Get and open the KYC Link which you can get from KYC or user endpoint: